wpimath/src/main/native/include/frc/fmt/Eigen.h

```diff
@@ -10,6 +10,10 @@
 #include <Eigen/SparseCore>
 #include <fmt/format.h>
 
+// FIXME: Doxygen gives internal inconsistency errors:
```

Comment:
Does this still happen with Doxygen 1.12.0?

Reply:
Yes